new-log-viewer/src/services/LogExportManager.ts

```diff
@@ -0,0 +1,42 @@
+import downloadDecompressedLogs from "../utils/downloadDecompressedLogs";
+
+
+class LogExportManager {
+    private blob: Blob;
```

Comment:
Can we use the "private class properties" (`#`) syntax? Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties